Django 2.2 test suite errors out when the selenium package
is not present. This way, it passes without any issue.

Also, due to temporary removal of usr/share/zoneinfo in 29259680,
one test is failing on SPARC (causing many others to fail as well).

Not intended for upstream.

--- Django-2.2.9/tests/admin_inlines/tests.py
+++ Django-2.2.9/tests/admin_inlines/tests.py
@@ -1,4 +1,4 @@
-from selenium.common.exceptions import NoSuchElementException
+#from selenium.common.exceptions import NoSuchElementException
 
 from django.contrib.admin import ModelAdmin, TabularInline
 from django.contrib.admin.helpers import InlineAdminForm
--- Django-2.2.11/tests/settings_tests/tests.py
+++ Django-2.2.11/tests/settings_tests/tests.py
@@ -320,6 +320,7 @@ class SettingsTests(SimpleTestCase):
 
     @requires_tz_support
     @mock.patch('django.conf.global_settings.TIME_ZONE', 'test')
+    @unittest.skipIf(sys.platform.startswith("sunos"), "Fails on SPARC due to 29259680")
     def test_incorrect_timezone(self):
         with self.assertRaisesMessage(ValueError, 'Incorrect timezone setting: test'):
             settings._setup()
